default.nix

```diff
@@ -6,4 +6,4 @@ if ! builtins ? nixVersion || builtins.compareVersions requiredVersion builtins.
 
 else
 
-  import ./pkgs/top-level
+  import ./pkgs/impure.nix
```

Comment:
I'm not sure about the location and name about this-one but don't have a better recommendation at the moment.